sdk/cosmosdb/cosmos/test/public/integration/multiregion.spec.ts

```diff
@@ -170,7 +170,7 @@ describe("Multi-region tests", function (this: Suite) {
         on: PluginOn.request,
         plugin: async (context) => {
```

Comment:
I think the `context` variable should be typed as [`RequestContext`](https://github.com/bzhang0/azure-sdk-for-js/blob/d95be117605d7fad2b6ff520cf5cd7ad314f3229/sdk/cosmosdb/cosmos/src/request/RequestContext.ts#L17) which will enforce the fact that `endpoint` is a string so there is no need to cast it explicitly.